examples/src/utils/MeasurementUi.ts

```diff
@@ -0,0 +1,71 @@
+import { Cognite3DViewer } from "@cognite/reveal";
+import { MeasurementLineOptions } from "@cognite/reveal/tools";
+import { MeasurementTool, MeasurementUnits } from "@cognite/reveal/tools";
+import dat from "dat.gui";
+
+export class MeasurementUi {
+  private readonly _viewer: Cognite3DViewer;
+  private readonly _measurementTool: MeasurementTool;
+  private _gui: dat.GUI;
+  private _guiController: any[];
+
+  private state = {
+    linewidth: 0.002,
+    color: '#00FFFF'
+  };
+
+  private types = {
```

Comment:
Maybe rename to something a little more descriptive, like guiData or something indicating what it is used for.